### PR TITLE
Update for 1.7.18

### PR DESCRIPTION
**Pull request by shimono**
Add Unit 3 Configuration Items

### CODE REVIEW

File: en/server-operator/unit_config_list.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-The default setting is defined in the "personium-unit-config-default.properties" file on the GitHub source.  
-Please do not change the default setting.  
-If you want to change the setting, place a java property file named "personium-unit-config.properties" and start the servlet container.  
-The unit then replaces the default value defined in "personium-unit-config-default.properties" with the value of each setting item defined in "personium-unit-config.properties".  
+The unit configuration file is a java property file named "personium-unit-config.properties" which manages the setting of Personium unit.  
+To configure a Personium unit, just place this file at a specified location and start the servlet container.  
+
+An unit uses the values of each setting items defined in "personium-unit-config.properties". It uses default values if not specified.
+
+The default setting is defined in the "personium-unit-config-default.properties" file on the GitHub source. Configuration of a unit can also be done by changing this file, but it is not recommended.
```
Comment (tochi-y) [inline en/server-operator/unit_config_list.md]: I think it should add '## Default settings' corresponding to the Japanese description.

File: en/server-operator/unit_config_list.md [hunk 2]
```diff
@@ -90,12 +97,16 @@ It is a setting which change from default as optional when operating Personium.
...
 |security.auth.password.scrypt.cpuCost|CPU cost of scrypt hash|Int<br>Power of 2|16384|core|v1.7.8 or later|
 |security.auth.password.scrypt.memoryCost|Memory cost of scrypt hash|Int|8|core|v1.7.8 or later|
 |security.auth.password.scrypt.parallelization|Number of parallelizations of scrypt hash|Int|1|core|v1.7.8 or later|
 |security.auth.password.scrypt.keyLength|Key length of scrypt hash|Int|32|core|v1.7.8 or later|
 |security.auth.password.scrypt.saltLength|Salt length of scrypt hash|Int|64|core|v1.7.8 or later|
+|security.auth.password.salt|Password hash salt value|16 hexadecimal character string|saltijkl|core|only applicable for SHA256 hash Algorithm|
+|security.token.defaultScope.ropc|default (maximum) scope given to the access tokens issued via ROPC process|Space separated scopes string|root|<br>|
+|security.token.defaultScope.assertion|default (maximum) scope given to the access tokens issued via assertion process|Space separated scopes string|root|<br>|
+|security.token.defaultScope.grant_code|default (maximum) scope given to the access tokens issued via grant_code process|Space separated scopes string|root|<br>|
```
Comment (tochi-y) [inline en/server-operator/unit_config_list.md]: Used component value is all blank. The value need to be 'core' or 'core,engine'.

File: ja/server-operator/unit_config_list.md [hunk 3]
```diff
@@ -97,6 +104,10 @@ Personiumを運用する上でデフォルトからの変更を任意として
 |security.auth.password.scrypt.parallelization|scryptハッシュの並列化数|Int|1|core|v1.7.8以降|
 |security.auth.password.scrypt.keyLength|scryptハッシュのキー長|Int|32|core|v1.7.8以降|
 |security.auth.password.scrypt.saltLength|scryptハッシュのソルト長|Int|64|core|v1.7.8以降|
+|security.auth.password.salt|SHA256アルゴリズム使用時のパスワードハッシュソルト値|16桁の16進文字列|saltijkl|core|scrypt使用時は無効|
+|security.token.defaultScope.ropc|ROPCでのアクセストークン発行時に付与されるデフォルト (最大) スコープ|スペース区切りのスコープ情報|root|<br>|
+|security.token.defaultScope.assertion| assertionでのアクセストークン発行時に付与されるデフォルト (最大) スコープ |スペース区切りのスコープ情報|root|<br>|
+|security.token.defaultScope.grant_code|grant_codeでのアクセストークン発行時に付与されるデフォルト (最大) スコープ|スペース区切りのスコープ情報|root|<br>|
```
Comment (tochi-y) [inline ja/server-operator/unit_config_list.md]: Used component value is all blank. The value need to be 'core' or 'core,engine'.

File: en/server-operator/unit_config_list.md (hunk 2 above)
Comment (tochi-y) [reply en/server-operator/unit_config_list.md]: It is better to add 'v1.7.18 or later' to the notes.

File: ja/server-operator/unit_config_list.md (hunk 3 above)
Comment (tochi-y) [reply ja/server-operator/unit_config_list.md]: It is better to add 'v1.7.18以降' to the notes.

File: ja/server-operator/unit_config_list.md [hunk 4]
```diff
@@ -97,6 +104,10 @@ Personiumを運用する上でデフォルトからの変更を任意として
 |security.auth.password.scrypt.parallelization|scryptハッシュの並列化数|Int|1|core|v1.7.8以降|
 |security.auth.password.scrypt.keyLength|scryptハッシュのキー長|Int|32|core|v1.7.8以降|
 |security.auth.password.scrypt.saltLength|scryptハッシュのソルト長|Int|64|core|v1.7.8以降|
+|security.auth.password.salt|SHA256アルゴリズム使用時のパスワードハッシュソルト値|16桁の16進文字列|saltijkl|core|scrypt使用時は無効|
+|security.token.defaultScope.ropc|ROPCでのアクセストークン発行時に付与されるデフォルト (最大) スコープ|スペース区切りのスコープ情報|root|<br>|
+|security.token.defaultScope.assertion| assertionでのアクセストークン発行時に付与されるデフォルト (最大) スコープ |スペース区切りのスコープ情報|root|<br>|
```
Comment (tochi-y) [inline ja/server-operator/unit_config_list.md]: Spaces should be removed.

before:

```
| assertionでのアクセストークン発行時に付与されるデフォルト (最大) スコープ |
```

after:

```
|assertionでのアクセストークン発行時に付与されるデフォルト (最大) スコープ|
```

**Comment by shimono** [comment]
Thank you for your reviewing. I made fixes.

**Pull request merged**